MICS6_Master.do

```diff
@@ -245,7 +252,9 @@ foreach name in $newMICS6countries {
 	drop if _merge == 2
 	drop _merge
 	gen country_name = "`name'"
-	
+	if inlist("`name'","Nepal2019") {
+	drop helevel2
```

Comment:
Could you clarify why dorp the variables here as we have the "keep" command at the very end that drop all the variables that are not needed?